src/transformers/modeling_utils.py

```diff
@@ -614,8 +614,8 @@ def generate(
         input_ids=None,
         max_length=None,
         min_length=None,
-        do_sample=True,
-        early_stopping=False,
+        do_sample=None,
```

Comment:
same as above

Reply:
I like it better like this, because all inputs are set no `None` . It defaults to `False` because that's how it's set in `self.config.do_sample` as a default. But up for debate I guess.

Reply:
You should document that in the docstring then. When someone sees "Defaults to" in the docstring and then gets different default behavior (because of their config) it would be confusing.